src/Component/AddPlaylist/AddPlaylist.jsx

```diff
@@ -1,12 +1,30 @@
 import { useState } from "react";
 import { usePlay } from "../../Context/Playlets-context/Playlets-context";
 import { useVideo } from "../../Context/Video-Context/video-context";
+import { useAuth } from "../../Context/Auth-context/Auth-context";
```

Comment:
you can use barrel exports for all contexts...

Reply:
will change it.